be/src/exec/aggregator.cpp

```diff
@@ -820,6 +819,51 @@ Status Aggregator::output_chunk_by_streaming(Chunk* input_chunk, ChunkPtr* chunk
     return Status::OK();
 }
 
+Status Aggregator::convert_to_spill_format(Chunk* input_chunk, ChunkPtr* chunk) {
+    auto use_intermediate_as_input = _use_intermediate_as_input();
```

Comment:
turn off query cache when spilling is on, or turn off spilling when query cache is on.  in FragmentExecutor::prepare